modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java

```diff
@@ -480,6 +483,24 @@ public DEDataRecordCollectionSaveResponse execute(
 		}
 	}
 
+	@Override
+	public DEDataRecordCollectionSearchResponse execute(
+			DEDataRecordCollectionSearchRequest deDataRecordCollectionSearchRequest)
+		throws DEDataRecordCollectionException {
+
+		DEDataRecordCollectionSearchExecutor
+			deDataRecordCollectionSearchExecutor =
+				getDeDataRecordCollectionSearchExecutor();
+
+		try {
+			return deDataRecordCollectionSearchExecutor.execute(
```

Comment:
@mclomello Change the variable
Sugestion:

```suggestion
			return _deDataRecordCollectionSearchExecutor.execute(
```

Reply:
Fixed according by previous comments.